### ISSUE TITLE
Pin Layout Issue

### ISSUE THREAD

**Issue by ehuegler** [QTC-UMD/dds-sweeper#38]
There is a memory misalignment issue which can cause the DDS to go into an unrecoverable error state. The current iteration of the board does not have any connection between the pico and the AD9959 reset pin that would reset the DDS, so the only way to recover is to power cycle the both the pico and the AD9959.

I did fix this problem by soldering an additional wire onto the boards. The problem is that to actually incorporate this wire as a trace on the breakout board is to switch the two of the GPIO pins the pico is currently using which can be done easily in the firmware.

Basically the problem can be trivially fixed but it would break the two working boards I have. For now these boards are still necessary to keep on testing. When I have a chance I just need to make new boards to replace them so that I can push the change to the repo.

**Comment by dihm** [comment]
Whenever one of us gets to this, it would be good to further widen some of those small traces winding around the Pico. We went to print another test board based on the files in the repo and they were still going to mill a bit too small.

**Comment by dihm** [comment]
This was fixed in #39
